src/main/java/xbony2/huesodewiki/HuesoDeWiki.java

```diff
@@ -74,59 +77,57 @@ public void preInit(FMLPreInitializationEvent event){
 	}
 	
 	private class RenderTickEventEventHanlder {
+		/**
+		 * @return The ItemStack that the player is currently hovering over. If they are hovering over an empty slot,
+		 * 		   are not hovering over a slot or they are hovering over a slot in a non-supported Gui, returns an
+		 * 		   empty ItemStack.
+		 */
+		@Nonnull
+		private ItemStack getHoveredItemStack(){
```

Comment:
Hmm, I think these would be better in the Utils class.

Reply:
I agree. I thought about it but wasn't totally sure.